modules/core/src/main/scala/org/scalasteward/core/edit/hooks/HookExecutor.scala

```diff
@@ -51,7 +51,7 @@ final class HookExecutor[F[_]](implicit
 
   private def execPostUpdateHook(repo: Repo, update: Update, hook: PostUpdateHook): F[EditAttempt] =
     for {
-      _ <- logger.info(s"Executing post-update hook for ${hook.groupId}:${hook.artifactId.name}")
+      _ <- logger.info(s"Executing post-update hook for ${update.groupId}:${update.mainArtifactId}")
```

Comment:
What do you think about adding the `command` to the log ? Currently we would see multiple lines for the same update stating `"Executing post-update hook for gid:ai"` without mentioning the executed hook.

Reply:
done